packages/ckeditor5-theme-lark/theme/ckeditor5-html-embed/htmlembed.css

```diff
@@ -8,6 +8,8 @@
 	--ck-html-embed-source-height: 10em;
 	--ck-html-embed-unfocused-outline-width: 1px;
 	--ck-html-embed-content-min-height: calc(var(--ck-icon-size) + var(--ck-spacing-standard));
+
+	--ck-html-embed-source-disabled-color: hsl(0deg 0% 45%)
```

Comment:
This variable is added to be consistent with not merged yet [https://github.com/ckeditor/ckeditor5/pull/8818/files,](https://github.com/ckeditor/ckeditor5/pull/8818/files,) which also fixed the contrast ratio.